src/mjolnir/util.cc

```diff
@@ -32,6 +32,7 @@ const std::string ways_file = "ways.bin";
 const std::string way_nodes_file = "way_nodes.bin";
 const std::string nodes_file = "nodes.bin";
 const std::string edges_file = "edges.bin";
+const std::string tile_manifest_file = "tile-manifest.json";
```

Comment:
everything else is `_`, we shoudl probably match that

Reply:
Yup fixing